### ISSUE TITLE
Capture sound from main channel

### ISSUE THREAD

**Issue by JelassTeid** [tomer8007/windows-shazam#2]
I try to do virtual micro with sound from main channel, but didn't it, it's crazy. Could u do capture from main channel(browser for e.x.), bot from micro?

**Comment by tomer8007** [comment]
I don't understand. What are you trying to do? What is virtual micro and what do you mean by main channel? 

The software just captures from the default microphone.

**Comment by JelassTeid** [comment]
Sorry for my bad eng maybe. I wanna  recognize music not from default microphone. I wanna recognize music from browser. Is it possible that programm listen sound from not micro. From headphones for e.x.

**Comment by tomer8007** [comment]
Oh, I understnand. I need to think of a way to isolate sound from the browser. Can you give me more details about your restrictions? For example, Is it practical for you to use the windows sound mixer for that?

**Comment by JelassTeid** [comment]
Yes ofc. If program will be listen music from default headphones/loudspeakers - will be awesome. I often watch twitch or youtube, and not all streamers talk about "what music play". 
![default](https://user-images.githubusercontent.com/40953122/42445119-8957ab0c-838b-11e8-8d80-210ce1cc5585.JPG)

**Comment by tomer8007** [comment]
Oh I think I get you.
I've always used this software to recognize music coming from the computer itself only.
That's why I [wrote in the wiki page](https://github.com/tomer8007/windows-shazam/wiki/Usage) how to configure your Windows so that the default "microphone" is in fact your sound card. At least it worked on my computer.

Does it work for you?

**Comment by weespin** [comment]
[Shockzam](https://github.com/weespin/Shockzam/) is a windows-shazam fork which aiming on recognizing from computer itself
